pkgs/development/python-modules/dwdwfsapi/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchPypi
+, requests
+, ciso8601
+}:
+
+buildPythonPackage rec {
+  pname = "dwdwfsapi";
+  version = "1.0.3";
+
+  disabled = pythonOlder "3.6";
```

Comment:
We don't support anything older than that anyway, so we can safely drop that.

Reply:
Updated, thanks.